### ISSUE TITLE
Support updating existing media in bulk

### ISSUE THREAD

**Issue by JakePT** [JakePT/automatic-alternative-text#8]
The plugin should support setting alt text on media in bulk. This should be relatively simple using Bulk Actions while in List mode, but adding a button to Grid mode seems to be more difficult.
